mobile/native/src/ln_dlc/mod.rs

```diff
@@ -510,7 +505,28 @@ fn keep_wallet_balance_and_history_up_to_date(node: &Node) -> Result<()> {
         })
     });
 
-    let trades = derive_trades_from_filled_orders()?;
+    let trades = db::get_all_trades()?;
```

Comment:
👍 nice, but I guess that will remove all open and close position entries from the history of existing users right?

Reply:
As discussed, updating to the new version will erase the trade history.